betfairlightweight/baseclient.py

```diff
@@ -100,26 +103,35 @@ def session_expired(self):
 
     @property
     def cert(self):
-        """Looks in /certs/ for betfair certs.
+        """The betfair certificates.
+
+        By default, it looks for the certificates in /certs/.
+
         :return: Path of cert files
         """
+        if self.cert_files is not None:
+            return self.cert_files
+
         certs = self.certs or '/certs/'
-        cert_paths = []
         ssl_path = os.path.join(os.pardir, certs)
         try:
             cert_path = os.listdir(ssl_path)
         except FileNotFoundError:
-            raise CertsError
+            raise CertsError(certs)
         except OSError:   # Python 2 compatability
-            raise CertsError
+            raise CertsError(certs)
 
+        cert = None
+        key = None
         for file in cert_path:
-            ext = file.rpartition('.')[2]
-            if ext in ['key', 'crt', 'pem']:
-                cert_path = ssl_path + file
-                cert_paths.append(cert_path)
-        cert_paths.sort()
-        return cert_paths
+            ext = os.path.splitext(file)[-1]
+            if ext in ['.crt', '.cert']:
+                cert = os.path.join(ssl_path, file)
+            elif ext == '.key':
+                key = os.path.join(ssl_path, file)
+        if cert is None or key is None:
+            raise CertsError(certs)
+        return [cert, key]
```

Comment:
`cert` and `key` as strings. In requests usually you pass them as tuple, but I return a list to maintain the same behaviour as before my change. Personally, I would change it to return `(cert, key)`.